portfolio/src/main/webapp/index.html

```diff
@@ -39,7 +40,7 @@ <h3>Social Profiles </h3>
         </p>
         
         <h3>Random Quotes</h3>
-      </div>
+        </div>
```

Comment:
Nit: adding the tab seems to make this unaligned with the opening div bracket. Please align.